### DISCUSSION TITLE
Ref Arch connecting to Third Party AWS peering requests

### DISCUSSION THREAD

**Discussion by moos3** [gruntwork-io/knowledge-base#391] [Help] [answered]
What is the best way to handle peering in AWS with third parties? ie. Fivetran's private link, which is just a vpc peering. As they only need access to Aurora in the given regions.

**Comment by yorinasub17** [comment, marked as answer]
Please see https://github.com/gruntwork-io/knowledge-base/discussions/107 which has a suggested way to do this.

Note that I have added some additional commentary to that entry regarding the recommended way to integrate the referenced module into the Reference Architecture.
